content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -85,43 +85,142 @@ func TestAccXxx(*testing.T)
+### GitHub Actions
+
+Create a [GitHub Actions workflow](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) file, such as `.github/workflows/test.yaml`, that:
+
+- Runs when pull requests are submitted (or on [other events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) as appropriate)
+- Uses [`actions/checkout`](https://github.com/actions/checkout) to checkout the provider codebase
+- Uses [`actions/setup-go`](https://github.com/actions/setup-go) to install Go
+- Uses [`hashicorp/setup-terraform`](https://github.com/hashicorp/setup-terraform) to install Terraform CLI
+- Runs the `go test` command with the appropriate environment variables and flags
+
+For example:
+
+```yaml
+name: Terraform Provider Tests
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/test.yaml'
+      - '**.go'
+
+permissions:
+  # Permission for checking out code
+  contents: read
+
+jobs:
+  acceptance:
+    name: Acceptance Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: '1.1.*'
+          terraform_wrapper: false
+      - run: go test -v -cover ./...
+        env:
+          TF_ACC: '1'
+  unit:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - run: go test -v -cover ./...
+```
 
-~> Note: When developing or testing Terraform plugins, we highly recommend
-running acceptance tests with an account dedicated to testing. This ensures no
-infrastructure is created or destroyed in error during development or validation
-of any Provider Resources in any environment that cannot be completely and
-safely destroyed.
+More advanced configuration, such as running acceptance testing against multiple Terraform CLI versions, is possible via the [`matrix`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) strategy.
+
+For example, adjusting the job configuration:
+
+```yaml
+jobs:
+  acceptance:
+    name: Acceptance Tests (Terraform ${{ matrix.terraform-version }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        terraform-version:
+          - '0.12.*'
+          - '0.13.*'
+          - '0.14.*'
+          - '0.15.*'
+          - '1.0.*'
+          - '1.1.*'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: ${{ matrix.terraform-version }}
+          terraform_wrapper: false
+      - run: go test -v -cover ./...
+        env:
+          TF_ACC: '1'
+```
 
 ## Environment Variables
```

Comment:
what would you think about having a table with three columns 
- variable name, default, description

Could make the formatting nicer to look at and make it more readable